src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const styleArray = sourceString.split(';');
+  const removingSpaces = styleArray.map((lines) => lines.trim());
+  const filterValues = removingSpaces.filter((elem) => elem !== '');
+
+  filterValues.forEach((elem) => {
+    const [key, value] = elem.split(':');
+
+    if (key && value) {
+      stylesObject[key.trim()] = value.trim();
```

Comment:
The check for `key && value` is good, but you might want to consider trimming `key` before the `if` statement to handle cases where the key might be an empty string after trimming, which should not be added to the `stylesObject`.